10/umbraco-deploy/deployment-workflow/deploying-changes.md

```diff
@@ -23,7 +23,7 @@ Here's a quick step-by-step on how you deploy these changes to your environment:
 
 ## Deploying without using a Git client
 
-If you don't have a Git client installed on your local machine, or prefer to work with Git through command line, you can use eg. Git Bash and the following commands:
+If you don't have a Git client installed on your local machine, you can use Git Bash for command-line Git operations. Execute the following commands:
```

Comment:
I believe it is best to keep Git as well and not just Git Bash and instead of "execute" to make it sound more friendly perhaps it can be changed with "do" or "run"?
```suggestion
If you don't have a Git client installed on your local machine, you can use Git or Git Bash for command-line Git operations. Run the following commands:
```